nucypher-core-python/nucypher_core/ferveo.pyi

```diff
@@ -1,21 +1,21 @@
-from typing import Sequence
-
+from typing import Sequence, final
 
+@final
 class Keypair:
     @staticmethod
     def random() -> Keypair:
         ...
 
     @staticmethod
-    def from_secure_randomness(data: bytes) -> Keypair:
+    def from_secure_randomness(bytes: bytes) -> Keypair:
```

Comment:
the name shadowing of the `bytes` function here strikes me as a bit unusual, even in the context of a stub.  I'd suggest sticking with some variant of `data` (`payload`, `secure_bytes`) or at minimum prepend with an underscore (`_bytes`.)

Do the parameter stubs need to be identical to the actual binding definition?  If so, then this will lead to the `bytes` function also being overloaded within the scope of this method.

Reply:
Yes, I got weirded out a little myself, but wasn't sure if I should comment on it :)